changelog/_unreleased/2024-10-26-Add-configurable-installer-timeout.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Add configurable installer timeout
+issue: NEXT-XXXXX
+author: Marc Christenfeldt
+author_email: marc.christenfeldt@gmx.de
+author_github: marcbln
+discussion: https://forum.shopware.com/t/shopware-update-bricht-wegen-zu-langer-laufzeit-ab/104013/11
+---
+
+# Core
+
+* Added new environment variable `SHOPWARE_INSTALLER_TIMEOUT` to configure the command timeout in the web installer
+* Added constant `DEFAULT_TIMEOUT` in `StreamedCommandResponseGenerator` to define the default timeout value of 900 seconds
+* Changed timeout handling in `StreamedCommandResponseGenerator` to support configurable timeout values from environment
+
+# Upgrade Information
+
+## Environment Configuration
+
+The web installer now supports configurable command timeouts through the environment variable `SHOPWARE_INSTALLER_TIMEOUT`. This value should be provided in seconds.
```

Comment:
We should mention that this needs to be inside `.env.installer` otherwise it won't get populated.

https://github.com/shopware/shopware/blob/2a6677ccbb177f397e8b770186a7d1c3fef159bd/src/WebInstaller/index.php#L11